lib/src/main/res/layout/list_item_color.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Yeah. I kinda have snippet that can calculate the contrast and change the text color accordingly. We could try that.